fabfile.py

```diff
@@ -326,12 +327,18 @@ def services_symlink_upstart(new_env):
         symlink_name
     ))
 
+def generate_secret_key():
+    chars = 'abcdefghijklmnopqrstuvwxyz0123456789!@#$%^&*(-_=+)'
+    return ''.join([
+        random.SystemRandom().choice(chars) for i in range(50)
+    ])
```

Comment:
Can't you https://github.com/django/django/blob/3c447b108ac70757001171f7a4791f493880bf5b/django/core/management/utils.py#L82 ?